.gitignore

```diff
@@ -1,10 +1,9 @@
-*~
-*.pyc
-
-.sass-cache/
+.DS_STORE
 _site/
-.bundle/
-
-# Ignore locked gems because GitHub Pages controls the gems in production, not Nuclide. Omit the
-# lock so `bundle install` always installs the latest.
-Gemfile.lock
```

Comment:
So we have changed our take on `Gemfile.lock` and actually **want** to lock a specific version of the Jekyll stack with each commit? Will this work on GIthub?